barretenberg/cpp/src/barretenberg/commitment_schemes_recursion/CMakeLists.txt

```diff
@@ -1 +1 @@
-barretenberg_module(commitment_schemes_recursion commitment_schemes stdlib_primitives)
\ No newline at end of file
+barretenberg_module(commitment_schemes_recursion commitment_schemes stdlib_poseidon2)
```

Comment:
why do we need stdlib poseidon and not primitives for commitment schemes?

Reply:
the module doesnt link without poseidon now that it's enabled and stdlib_poseidon2 already includes stdlib_primitives